keyboards/ymd96/ymd96.h

```diff
@@ -41,6 +41,19 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 	{      K700,       K701,       K702,       K703,       K704,       K705,       K706,       KC_NO,      K708,       K709,       K710,       K711,       K712,       K713,      K714  } \
 }
 
+#define KEYMAP_JJ50( \
```

Comment:
Could you rename `KEYMAP_JJ50` to `LAYOUT_jj50`to adhere to the new QMK standards.